apps/rstudio/form.yml.erb

```diff
@@ -34,10 +29,7 @@ attributes:
 form:
   - auto_queues
   - toolchain
-  - r_bundle_cran
-  - r_bundle_bioconductor
   - global_num_cores
-  - bc_num_slots
```

Comment:
I would like to keep the forms as uniform as possible. That's the reason why it's here although in most cases it will not be useful.

Reply:
The plan is indeed to have some type of foldable panel with advanced options, and keep what is displayed for all apps by default to a bare minimum with just the most common options.

We still need to define how this advanced panel will be done in the forms.